volcanic-alert-levels/VAL_WhakaariWhiteIsland.csv

```diff
@@ -39,4 +39,5 @@ PeriodStart UTC,PeriodEnd UTC,VAL,VAL System Version,Description,PeriodStart Loc
 2024-05-23T22:45:00Z,2024-05-24T04:59:59Z,3,3,Minor volcanic eruption,2024-05-24 10:45:00 NZST,2024-05-24 16:59:59 NZST
 2024-05-24T05:00:00Z,2024-05-24T23:44:59Z,2,3,Moderate to heightened volcanic unrest,2024-05-24 17:00:00 NZST,2024-05-25 11:44:59 NZST
 2024-05-24T23:45:00Z,2024-05-29T02:59:59Z,3,3,Minor volcanic eruption,2024-05-25 11:45:00 NZST,2024-05-29 14:59:59 NZST
-2024-05-29T03:00:00Z,9999-01-01T00:00:00Z,2,3,Moderate to heightened volcanic unrest,2024-05-29 15:00:00 NZST,9999-01-01 00:00:00 NZST
+2024-05-29T03:00:00Z,2024-08-20T04:59:59Z,2,3,Moderate to heightened volcanic unrest,2024-05-29 15:00:00 NZST,2024-08-20 16:59:59 NZST
+2024-08-20T05:00:00Z,9999-01-01T00:00:00Z,3,3,Minor volcanic eruption,2024-08-20T17:00:00 NZST,9999-01-01 00:00:00 NZST
```

Comment:
start time should be 
`2024-08-09 17:00:00 NZST` and `2024-08-09T05:00:00` UTC

Reply:
How did I make that mistake? I didn't even realise its a week in the future! Thankyou

Reply:
haha it happens to the best! <3